api/autopilot.go

```diff
@@ -140,6 +140,10 @@ func (sb HostScoreBreakdown) String() string {
 	return fmt.Sprintf("Age: %v, Col: %v, Int: %v, SR: %v, UT: %v, V: %v, Pr: %v", sb.Age, sb.Collateral, sb.Interactions, sb.StorageRemaining, sb.Uptime, sb.Version, sb.Prices)
 }
 
+func (hgb HostGougingBreakdown) DownloadGouging() bool {
```

Comment:
leaving comment to prevent merge - should extend UI to make sure we can set the surcharge multiplier